Documentation/kube-flannel.yml

```diff
@@ -106,6 +106,7 @@ data:
   cni-conf.json: |
     {
       "name": "cbr0",
+      "cniVersion": "0.3.1",
```

Comment:
I guess this has to be added to the other yml files as well, right?

Reply:
you are right I do it can you please take another look?